lib/debugletapi.js

```diff
@@ -84,14 +84,11 @@ DebugletApi.prototype.init = function(uid, logger, callback) {
     });
   }
 
-  if (process.env.GCLOUD_PROJECT_NUM) {
-    var projectNumber = process.env.GCLOUD_PROJECT_NUM;
-    var err = /^\d+$/.test(projectNumber) ?
-      null :
-      new Error('GCLOUD_PROJECT_NUM should be numeric');
+  if (process.env.GCLOUD_PROJECT) {
+    var projectNumber = process.env.GCLOUD_PROJECT;
```

Comment:
rename variables related to this to `project` and `project_` and update the jsdoc for `this.project_`.